src/app/components/Video.js

```diff
@@ -9,31 +9,43 @@ export default class Video extends Component {
     })).isRequired,
   };
 
+  static defaultProps = {
+    showMuteButton: true,
+  };
+
   #container = createRef();
   #video = createRef();
   #observer = null;
 
   state = {
     muted: true,
+    playing: false,
+    showPlayButton: this.props.showPlayButton
   };
 
   componentDidMount () {
-    setTimeout(() => {
-      this.#observer = new IntersectionObserver(this.#handleEntry, {
-        root: document.querySelector('.Screen'),
-        rootMargin: '0px',
-        threshold: 0.01,
-      });
-      this.#observer.observe(this.#container.current);
-    }, 2000);
+    if(!this.props.poster){
+      setTimeout(() => {
```

Comment:
Store timeout id in `#timeout`, then clear the timeout on `componentWillUnmount`